pages/common/git.md

```diff
@@ -2,18 +2,26 @@
 
 > Distributed version control system.
 
-- Check the Git version:
+- Show file changes:
 
-`git --version`
+`git status`
 
-- Call general help:
+- Track file changes:
 
-`git --help`
+`git add {{file}}`
 
-- Call help on a command:
+- Commit file changes:
 
-`git help {{command}}`
+`git commit -m "{{message}}"`
 
-- Execute Git command:
+- Upload changes to remote server:
 
-`git {{command}}`
+`git push {{server name}} {{branch}}`
```

Comment:
I'd recommend changing "server name" to "repository name" for more clarity. "remote repository" could work as well.

Reply:
I'll change it to "remote repository" to introduce a dash of the official Git terminalolgy.